rails/Gemfile.lock

```diff
@@ -367,4 +367,4 @@ RUBY VERSION
    ruby 2.5.1p57
 
 BUNDLED WITH
-   1.17.1
+   2.0.1
```

Comment:
Maybe commit the change to revert this bundler vesion?

Reply:
Done 😎